src/bot/setup.js

```diff
@@ -136,7 +145,7 @@ const getInitialoutAmount = async (
 			inputMint: new PublicKey(inputToken.address),
 			outputMint: new PublicKey(outputToken.address),
 			amount: amountToTrade,
```

Comment:
jup-ag v4 requires a BigInt for the amount.

Reply:
@arbcaptain @alexonab I don't see the change, but if it's here, plz approve